runtime-modules/membership/src/lib.rs

```diff
@@ -489,6 +536,149 @@ decl_module! {
 
             Self::deposit_event(RawEvent::ReferralCutUpdated(value));
         }
+
+        /// Transfers invites from one member to another.
+        #[weight = 10_000_000] // TODO: adjust weight
+        pub fn transfer_invites(
+            origin,
+            source_member_id: T::MemberId,
+            target_member_id: T::MemberId,
+            number_of_invites: u32
+        ) {
+            Self::ensure_member_controller_account_signed(origin, &source_member_id)?;
```

Comment:
Shouldn't this be returning the membership, the second call to `ensure_membership` should be avoided.

Reply:
Yes, you're correct. Fixed.